modules/project/variables.tf

```diff
@@ -208,6 +208,23 @@ variable "project_create" {
   default     = true
 }
 
+variable "service_agent_encryption_key_ids" {
+  description = "Service Agents to be granted encryption/decryption permissions over Cloud KMS encryption keys. Format {SERVICE_AGENT => [KEY_ID]}."
+  type        = map(list(string))
```

Comment:
Is it worth doing this with a SERVICE key, and resolving internally? Basically what we had before. I am doing this with actual large customers, and for them what matters is the service, not the service agent which is an internal implementation detail from their PoV.

Reply:
discussed with Julio, this will change a few things so we will postpone at scale and only care about a few critical data services